js/src/framework.test.ts

```diff
@@ -547,4 +551,10 @@ test("Eval with noSendLogs: true runs locally without creating experiment", asyn
   expect(result.summary.experimentUrl).toBeUndefined();
   expect(result.summary.scores.exact_match.score).toBe(1);
   expect(result.summary.scores.simple_scorer.score).toBe(0.8);
+
+  // Most importantly: verify that flush was never called (no logs sent)
+  expect(flushSpy).not.toHaveBeenCalled();
```

Comment:
sorry for being picky .. could we use https://github.com/braintrustdata/braintrust-sdk/blob/445cba50d36e46a64605ecbca35a76c94a528db3/js/src/logger.test.ts#L21 

then 

```
  await memoryLogger.flush();
  expect(await memoryLogger.drain()).length(0);
```